// api/mockProducts.js
export default [
  {
    id: "1",
    categoryId: "1",
    name: "The Programming for beginners",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 823,
    image: "/uploads/iphone5c-selection-hero-2013.png",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "2",
    categoryId: "1",
    name: "Python for Dummies",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 953,
    image: "/uploads/51u6y9Rm8QL._SY300_.jpg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "3",
    categoryId: "4",
    name: "How to master JavaScript in 10 minutes",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 764,
    image: "/uploads/_35%20(1).JPG",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "4",
    categoryId: "5",
    name: "How to pass programming interviews",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 674,
    image: "/uploads/Lumia1520-Front-Back-png.png",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "5",
    categoryId: "3",
    name: "10 techniques to master machine learning",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 674,
    image: "/uploads/htc-one-m7-802w-dual-sim-silver.jpg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  },
  {
    id: "6",
    categoryId: "2",
    name: "Data Science and its future",
    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
    price: 674,
    image: "/uploads/Agnes_Case_for_Samsung_Galaxy_S6_(1)__92643_thumb.jpg",
    cpu: "1.3GHz Apple A6",
    camera: "8mp (3264x2448)",
    size: "124.4mm x 59.2mm x 8.97mm (4.9 x 2.33 x 0.35)",
    weight: "132 grams (4.7 ounces) with battery",
    display: "4.0 326 pixel density",
    battery: "1480 mAh",
    memory: "16GB, 32GB and RAM 1 GB"
  }
]
